projects/canopy/src/lib/brand-icon/brand-icon.component.ts

```diff
@@ -76,9 +76,18 @@ export class LgBrandIconComponent {
    * incorrect brand-icon.
    */
   private setSVGAttributes(svgData: string): string {
+    let idCount = 0;
+    let xlinkCount = 0;
+
     return svgData
-      .replace(/id="\w+"/g, () => `id="lg-brand-icon-${this.id}"`)
-      .replace(/xlink:href="#\w+"/g, () => `xlink:href="#lg-brand-icon-${this.id}"`);
+      .replace(/id="([^"]+)"/g, () => {
+        idCount++;
+        return `id="lg-brand-icon-${this.id}-${idCount}"`;
+      })
```

Comment:
```suggestion
      .replace(/id="([^"]+)"/g, () =>
        `id="lg-brand-icon-${this.id}-${idCount++}"`)
```
It's style only but you could do this (if readability is still ok?)

Reply:
yeah that's a bit slicker. made that change